templates/tickets/moderation_assigned.html

```diff
@@ -42,74 +43,142 @@
         loadUrl = '{% url "tickets-user-annotations" 0 %}'.replace('0', ''+uploader_id);
         $('#user-annotations-section').load(loadUrl);
     };
+
+    // Function to play the sound corresponding to a selected ticket
+    var soundPlayerLoadedCheck;
+    function playSoundFromTicketList(soundID){
+        // Because when this function is called the sound player is still not loaded in the HTML, we use a setInterval
+        // function that will keep onchecking if the sound player has been loaded. Once player is loaded, this
+        // function will clear the interval and play the sound.
+        soundPlayerLoadedCheck = setInterval(function(){
+            var playerPlayButton = $("#sound_" + soundID + " a.toggle.play");
+            if (playerPlayButton.length){
+                console.log(playerPlayButton);
+                clearInterval(soundPlayerLoadedCheck);
+                playerPlayButton[0].click();
+            }
+        }, 100);
+    }
  
     var selectAllOnClick = true;
     $(document).ready(function() {
 
         {% if moderator_tickets_count %}
 
-        $("#select-all").click(function(){
-            if (selectAllOnClick){
-                $("#select-all").html("select none");
-            } else {
-                $("#select-all").html("select all");
-            }
-            $('.ticket-check').attr('checked', selectAllOnClick);
-            selectAllOnClick = !selectAllOnClick;
-            updateSelectedTickets();
-        });
+            // Bind submit form event to add automplay and  include deferred options as query strings
+            var moderationFormElement = $("#moderation-form").find('form')[0];
+            $(moderationFormElement).submit(function(){
+                var includeDeferred = document.getElementById("select_deferred_option").checked;
+                var shouldAutoplay = document.getElementById("select_autoplay_option").checked;
+                var newFormAction = '.?';
+                if (includeDeferred){ newFormAction += 'include_d=on&' }
+                if (shouldAutoplay){ newFormAction += 'autoplay=on' }
```

Comment:
it might be nicer to use an object here and then parse it to [jquery.param](https://api.jquery.com/jquery.param/) or [urlsearchparams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams)

Reply:
nice trick, I did not know about it :)